Makefile

```diff
@@ -1,6 +1,8 @@
 PROJECT_NAME:=senfd
 PROJECT_VERSION:=0.2.3
 
+BROWSER:=chromium-browser
+
```

Comment:
Its not perfect, but would it not be better to use something mere generic such as `xdg-open`?

Reply:
Yeah, but it only kinda-sorta works anyway... great on Linux but not elsewhere...